include/cling/Interpreter/Interpreter.h

```diff
@@ -613,6 +613,7 @@ namespace cling {
 
     clang::CompilerInstance* getCI() const;
     clang::Sema& getSema() const;
+    IncrementalParser& getIncrParser() const { return *m_IncrParser; }
```

Comment:
Vassil feels fairly strongly about this (I don't), so for now I'd appreciate if you could remove this interface.

The goal behind this is not to reduce the interface size. It's to encourage separation of parsing and codegen and jitting, by making cross-talk more difficult.